src/app/settings/settings.ts

```diff
@@ -78,7 +78,7 @@ export class Settings {
   // How many columns to display vault buckets
   vaultMaxCol = 999;
   // How big in pixels to draw items - start smaller for iPad
-  itemSize = window.matchMedia('(max-width: 1025px)').matches ? 38 : 44;
+  itemSize = window.matchMedia('(max-width: 1025px)').matches ? 38 : 48;
```

Comment:
Long term I think we should remove the ability to customize the item tile size and choose a size appropriate to the window size. What do you think? (not a blocking comment)

Reply:
Yeah.  How does it feel on devices that are so much bigger?  I have a iPhone 6 so that's not big...  Wife has a 7.  Now there are these 10 S Max 6.5" displays.  Sigh.